internal/compat/js_table.go

```diff
@@ -392,7 +392,6 @@ var jsTable = map[JSFeature]map[Engine][]versionRange{
 		Edge:    {{start: v{14, 0, 0}}},
 		ES:      {{start: v{2015, 0, 0}}},
 		Firefox: {{start: v{51, 0, 0}}},
-		Hermes:  {{start: v{0, 7, 0}}},
```

Comment:
This file is generated by "compat-table.js". 

I think that the way to go is add `hermes: false` here
https://github.com/evanw/esbuild/blob/71be8bc24e70609ab50a80e90a17a1f5770c89b5/scripts/compat-table.js#L145